content/docs/ca/contributing/documentation.md

```diff
@@ -1,2 +1,33 @@
 # Contribute to this documentation
 This documentation is a work-in-progress and *you are welcome to pitch in and help*. Simply fork the [Drupal Console Book](https://github.com/hechoendrupal/drupal-console-book "(i.e. this documentation)") on GitHub. If you haven't yet contributed to a project on GitHub, or aren't still sure what the workflow looks like, read the documentation about [pull requests](https://help.github.com/articles/using-pull-requests/). You may also wish to download the GitHub application ([Mac](https://mac.github.com) | [Windows](https://windows.github.com), which simplifies the workflow a bit and provides a nice GUI for your contributions).
+
+*This page is a meeting point for translators. If you're one, consider yourself invited to edit this page and state constancy of your opinions and your work*
+
+# Guide for translators
+If you would like to help in the translation of this book, consider yourself welcome. Please, keep in mind the following guidelines:
+
+* Be gender neutral.
+* Code and examples should not be translated.
+* Use a formal style. We prefer a formal tone.
+
+# Generate translated all available commands
+*Over time, DrupalConsole programmers add more and more commands, so it's possible some available commands weren't present in this book if they are not added to the book as they are added.
+In order to solve this problem, with a single command is possible to generate (already translated) all the .json files belonging to all the available commands*
+
+##Comand for generating documentation of all available commands currently
+If you have cloned in your local machine both projects, DrupalConsole and this book, you should have this [console-develop](https://github.com/weknowinc/drupal-console-develop) repository and execute the following command from a directory where Drupal were installed (or using the *--root* option):
+
+``` drupal develop:doc:data --path=/path/directorio/drupalconsole.com/content/data/commands-ca.json ```
```

Comment:
@harold20 This should be 
``` 
drupal develop:doc:data --path=/path/directorio/drupalconsole.com/content/data/commands-ca.json 
```

Reply:
What is wrong here? it is the same @jmolivas

Reply:
Actually is like this 
``` drupal develop:doc:data --path=/path/directorio/drupalconsole.com/content/data/commands-ca.json ```

I am asking to change as this
``` 
drupal develop:doc:data --path=/path/directorio/drupalconsole.com/content/data/commands-ca.json 
```

Add new lines